tools/bin/hawqregister

```diff
@@ -469,8 +481,8 @@ class HawqRegister(object):
             self.filepath = self.files[0][:self.files[0].rfind('/')] if self.files else ''
             check_file_not_folder()
             check_database_encoding()
-            if self.mode != 'force' and self.mode != 'repair':
-                if not create_table():
+            if self.mode != 'repair':
+                if not create_table() and self.mode != 'force':
```

Comment:
A little confused here, I think handle the edge case in _init_mode() could be much more clear. In detail, in this case set self.mode='usage2_table_non_exist' instead of 'force'.